client/gui/translations/ws_desktop_de.ts

```diff
@@ -9,7 +9,7 @@
     </message>
     <message>
         <source>Clear</source>
-        <translation>Klar</translation>
+        <translation>leeren</translation>
```

Comment:
it was "Clear" for klar like a cleaned window but not a emtptied text box / one you can empty with that button.